src/uipath/_cli/_interactive/_execution.py

```diff
@@ -0,0 +1,153 @@
+"""Execution utilities for running evaluations."""
+# type: ignore
+
+import subprocess
+import sys
+from pathlib import Path
+from typing import TYPE_CHECKING, Optional
+
+from .._utils._console import ConsoleLogger
+
+if TYPE_CHECKING:
```

Comment:
TYPE_CHECKING is a band-aid for bad architecture. If you have circular dependencies, the real fix is to restructure the code, not work around it with import tricks.